Runtime/AvatarCreator/WebRequests/AvatarAPIRequests.cs

```diff
@@ -118,7 +118,7 @@ public async Task<AvatarProperties> GetAvatarMetadata(string avatarId)
             var response = await authorizedRequest.SendRequest<Response>(
                 new RequestData
                 {
-                    Url = AvatarEndpoints.GetAvatarMetadataEndpoint(avatarId),
+                    Url = $"{Env.RPM_API_V2_BASE_URL}avatars/{avatarId}.json",
```

Comment:
now I actually see it in action I don't particularly love it. I think it becomes harder to maintain and more error prone. 
If you look at this particular url part  
`{Env.RPM_API_V2_BASE_URL}avatars/`
it's kinda copy pasta everywhere and also done so inconsistently. Some have the "/" before the avatars, others have "/" after it. 
![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/3c491e06-0653-4074-9823-cd176ee6a594)

One thing that may at least help here is making a constant 
EG
```cs
const string V2_Avatars = $"{Env.RPM_API_V2_BASE_URL}avatars/"
```

Reply:
Yeah, I only did it quickly for the draft PR purposes, but would need to check and test the links before making it not a draft PR 🙏

I wouldn't be against having a constant at the top of the class for the base url which pertains to the class that it's in. What makes you think it's harder to maintain?

Endpoint URLs pretty much never change, so for something so static I don't understand what the maintenance issue would be 🤔

Reply:
Yup, constants at the tops of the files make sense - added 👍